inc/customizer/class-astra-fonts.php

```diff
@@ -112,7 +112,20 @@ public static function render_fonts() {
 		}
 
 		$google_font_url = self::google_fonts_url( $google_fonts, $font_subset );
-		wp_enqueue_style( 'astra-google-fonts', $google_font_url, array(), ASTRA_THEME_VERSION, 'all' );
+
+		/**
+		 * Support self hosted Google Fonts.
+		 *
+		 * @since x.x.x
+		 */
+		if ( astra_get_option( 'load-google-fonts-locally', false ) && ! is_customize_preview() ) {
```

Comment:
```suggestion
		if ( astra_get_option( 'load-google-fonts-locally' ) && ! is_customize_preview() ) {
```
Default is already defined.

Reply:
Correct, Done 👍